<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1160</width>
    <height>791</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;黑体&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="main_horizontal_layout">
      <item>
       <layout class="QVBoxLayout" name="left_vertical_layout">
        <item>
         <widget class="QListWidget" name="switch_stack_list_widget"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="info_text_browser"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="right_stacked_widget">
        <widget class="QWidget" name="main_page">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="main_page_vertical_layout">
            <item>
             <layout class="QHBoxLayout" name="main_page_pyvista_horizontal_layout">
              <item>
               <widget class="QGroupBox" name="main_page_flight_test_group_box">
                <property name="title">
                 <string>试飞模型</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QGroupBox" name="main_page_simulation_group_box">
                <property name="title">
                 <string>仿真模型</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTabWidget" name="main_page_tab_widget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="main_page_time_domain_matplotlib_tab">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <attribute name="title">
                <string>Tab 1</string>
               </attribute>
              </widget>
              <widget class="QWidget" name="main_page_frequency_domain_matplotlib_tab">
               <property name="font">
                <font>
                 <family>黑体</family>
                 <pointsize>12</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <attribute name="title">
                <string>Tab 2</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="signal_page"/>
        <widget class="QWidget" name="simulation_page"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1160</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Domains"/>
    <addaction name="actionLoad_Simulation_Database"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTool">
    <property name="title">
     <string>Tool</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuOption"/>
   <addaction name="menuTool"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Domains">
   <property name="text">
    <string>Load Domains</string>
   </property>
  </action>
  <action name="actionLoad_Simulation_Database">
   <property name="text">
    <string>Load Simulation Database</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
